packages/alfa-selector/test/selector.spec.tsx

```diff
@@ -692,6 +692,20 @@ test(".parse() parses a named pseudo-class selector", (t) => {
   });
 });
 
+test(".parse() parses :host pseudo-class selector", (t) => {
+  const parseResult = parse(":host");
+
+  if (parseResult.isErr()) {
+    t.fail(parseResult.getErr());
+    return;
+  }
+
+  t.deepEqual(parseResult.get().toJSON(), {
```

Comment:
```suggestion
  t.deepEqual(parse(":host").get().toJSON(), {
```
We don't really need the first test. If the test throw an exception, that's almost as good as failing. Other tests in this file seem to use this pattern.